packages/manager/cypress/e2e/core/linodes/rebuild-linode.spec.ts

```diff
@@ -386,8 +386,10 @@ describe('rebuild linode', () => {
       // Type a root password
       assertPasswordComplexity(rootPassword, 'Good');
 
-      // Open the User Data accordion
-      ui.accordionHeading.findByTitle('Add User Data').scrollIntoView().click();
+      // Verify a into notice shows because this Linode has existing user data
```

Comment:
Little typo in comment:
```suggestion
      // Verify a info notice shows because this Linode has existing user data
```

Reply:
Good catch. Thank you!